src/applications/disability-benefits/686/config/form.js

```diff
@@ -173,6 +168,23 @@ const formConfig = {
               }
             }
           }
+        },
+        applicantInformationConditional: {
```

Comment:
We could call this `claimantInformation`, and have the path below be `claimant-information`.

Reply:
k